README.md

```diff
@@ -36,6 +36,23 @@ have to define the following `.mvn/extensions.xml` file:
 
 The download from Maven Central will be done by Maven itself.
 
+This extension allows printing the report to a JSON file instead of the stdout. To enable this you  
+simply have to set the property `<maven-buildtime-profiler-output>` with value `json`  
+under `<properties>` section in your project's pom.
+
+Additionally, you can set the property `<maven-buildtime-profiler-directory>` to indicate  
+the destination folder of the `report.json` file:
+
+```xml
+  <properties>
```

Comment:
@Paleloser Could we move these properties to extension parameters? Is that possible?

Reply:
I believe maven extensions don't allow parameters, as they don't work with Mojos: http://maven.apache.org/examples/maven-3-lifecycle-extensions.html

Reply:
Technically you could use a system property ....?

Reply:
Resolved at 7e586f1d27802bdafb5504095f5002ba515bc941.